lib/CSkillHandler.h

```diff
@@ -70,6 +70,7 @@ class DLL_LINKAGE CSkill : public Skill
 	LevelInfo & at(int level);
 
 	std::string toString() const;
+	bool obligatory(bool minor) const { return minor ? obligatoryMinor : obligatoryMajor; };

```

Comment:
Make two methods instead. Or use enum class instead of bool
`obligatory(false)` is not exactly self-describing.

Or (maybe even better) rename "obligatory skills" to "skill type" / "skill group".
We will have two skill groups:
- wisdom group (that consists from single skill)
- magic school group (with 4 skills in it)

And each of these group would have (at least for now) hardcoded mechanics - proposed at specific levels.

Reply:
done as enum class.